### PR TITLE
DataBox Edge - Rename from edgegateway to databoxedge, added a new version of API

### PR DESCRIPTION
**Pull request by anponnet**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [x] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [x] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [x] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

File: specification/databoxedge/resource-manager/Microsoft.DataBoxEdge/stable/2019-07-01/databoxedge.json [hunk 1]
```diff
@@ -0,0 +1,4125 @@
...
+            "description": "Error response describing why the operation failed.",
+            "schema": { "$ref": "#/definitions/CloudError" }
+          }
+        },
+        "x-ms-examples": {
+          "UploadCertificatePost": {
+            "$ref": "./examples/UploadCertificatePost.json"
+          }
+        }
+
```
Comment (nschonni) [inline specification/databoxedge/resource-manager/Microsoft.DataBoxEdge/stable/2019-07-01/databoxedge.json]: Fulfilment -> Fulfillment
```suggestion
          "enum": [ "Untracked", "AwaitingFulfillment", "AwaitingPreparation", "AwaitingShipment", "Shipped", "Arriving", "Delivered", "ReplacementRequested", "LostDevice", "Declined", "ReturnInitiated", "AwaitingReturnShipment", "ShippedBack", "CollectedAtMicrosoft" ],
```

**Review by nschonni — COMMENTED** [review]
You'll need to update the ignore for the misspelled "Fulfilment" for the old version if you're renaming the file
https://github.com/Azure/azure-rest-api-specs/blob/1cf40364e6a4456ad2f6360656864b1eed11cf0b/cSpell.json#L572-L576

**Comment by lirenhe** [comment]
@anponnet , there are a couple of tool check errors in this PR, could you help to fix the Avocado error which is required for the merge (public.rest-api-specs (Avocado) Failing after 6m — Avocado failed )?

**Review by KrisBash — APPROVED** [review]
Reviewed changes in the new version vs 2019-03-01. LGTM

**Comment by lirenhe** [comment]
@anponnet , please take a look at the tool validation error in https://dev.azure.com/azure-sdk/public/_build/results?buildId=110606&view=logs&j=f10f5372-40ed-54fc-caa4-b1cb5bd1252b  and https://dev.azure.com/azure-sdk/public/_build/results?buildId=110606&view=logs&j=8d2f9c49-cd83-5a9d-f3fe-2fd651afa742

**Comment by anponnet** [comment]
> @anponnet , please take a look at the tool validation error in https://dev.azure.com/azure-sdk/public/_build/results?buildId=110606&view=logs&j=f10f5372-40ed-54fc-caa4-b1cb5bd1252b  and https://dev.azure.com/azure-sdk/public/_build/results?buildId=110606&view=logs&j=8d2f9c49-cd83-5a9d-f3fe-2fd651afa742

@lirenhe I have fixed the validation errors and tests are passed. Can you please review

**Review by lirenhe — APPROVED** [review]

**Comment by nschonni** [comment]
@lirenhe the spellcheck was flagging a typo in the Enum value. It was suppressed on the old version (but not fixed with the rename). Not sure the new version should be merged/released without fixing that

**Comment by NullMDR** [comment]
@anponnet Would you please fix the spellcheck error https://github.com/Azure/azure-rest-api-specs/pull/7115/checks?check_run_id=223623035

**Comment by anponnet** [comment]
> @anponnet Would you please fix the spellcheck error https://github.com/Azure/azure-rest-api-specs/pull/7115/checks?check_run_id=223623035

@Phoenix We have to release a new version of API to fix it. We will fix it and send a new CR with updated version

**Pull request merged**